web-server/src/types/resources.ts

```diff
@@ -664,15 +664,23 @@ export enum AiSummarySource {
   PRS_ANALYSIS = 'PRS_ANALYSIS'
 }
 
-export type OrgSettings = {
+export type OrgAlertSettings = {
```

Comment:
@Kamlesh72 Although not in this PR,  but would you to open a subsequent PR to  completely remove the usage of OrgAlertSettings. Its not being used in the backend anymore. Also, longer names might sound verbose but something like `OrgShouldSyncAlertsAsIncidentsSetting` would have been a better name change, makes it more obvious.

Reply:
Ok sure I will make the PR. 
I will keep in mind for better naming.